src/i_video.c

```diff
@@ -1729,6 +1729,12 @@ void I_InitGraphics(void)
     ResetResolution(CurrentResolutionHeight(), true);
     CreateSurfaces(video.pitch, video.height);
     ResetLogicalSize();
+
+    // clear out any events waiting at the start and center the mouse
+    SDL_Event dummy;
+    while (SDL_PollEvent(&dummy))
+        ;
+    I_ResetRelativeMouseState();
```

Comment:
This seems safer so other important events aren't discarded:

```c
    SDL_PumpEvents();
    SDL_FlushEvent(SDL_MOUSEMOTION);
    I_ResetRelativeMouseState();
```

Reply:
But it still doesn't help with KDE Plasma?

Reply:
No it's the same behavior but we aren't discarding events like "gamepad connected".

Reply:
> But it still doesn't help with KDE Plasma?

And to be clear, the fix still helps the original issue so it no longer instantly rotates view when the mouse isn't even moved.

Reply:
> This seems safer so other important events aren't discarded:

Done in https://github.com/fabiangreffrath/woof/pull/1596/commits/1d6d3d59e06bd6b59b16aa3fdf8342a87a1aa451